tests/utils_tests/parsing_test.go

```diff
@@ -49,10 +49,10 @@ func TestParsePrefixedConceptIdsAndDichotomousIds(t *testing.T) {
 	requestContext.Request = new(http.Request)
 	requestBody := "{\"variables\":[{\"variable_type\": \"concept\", \"concept_id\": 2000000324}," +
 		"{\"variable_type\": \"concept\", \"concept_id\": 2000000123}," +
-		"{\"variable_type\": \"custom_dichotomous\", \"cohort_ids\": [1, 3]}]}"
+		"{\"variable_type\": \"custom_dichotomous\", \"provided_name\": \"test\", \"cohort_ids\": [1, 3]}]}"
```

Comment:
can we add another custom diachotomous variable but with no provided_name to test out the auto generate name?

Reply:
actually nvm saw we're testing it in TestRetrieveAttritionTable